### PR TITLE
How ClickHouse helps us track billions of GraphQL requests monthly

### PR DESCRIPTION
**Pull request by kamilkisiela**
it’s long and boring but I wanted to write something that is technical enough to help anyone understand how the usage reporting works and to learn the entire data pipeline.

### CODE REVIEW

**Comment by kamilkisiela** [comment]
@charlypoly any help with the thumbnail?

File: website/pages/blog/graphql-hive-and-clickhouse.mdx [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+tags: [graphql, graphql-hive]
+authors: [kamil]
+date: 2022-11-02
+description: Learn how ClickHouse enabled GraphQL Hive to scale from millions to billions of requests monthly.
+image: /blog-assets/announcing-graphql-hive/cover.png
+thumbnail: /blog-assets/announcing-graphql-hive/thumbnail.png
+---
+
+You will learn how **ClickHouse** and **Kafka** enabled GraphQL Hive to collect billions of requests monthly without a drop of sweat.
+We will also cover the challenges we faced and how we overcame them. The article explains the architecture of our system in detail.
```
Comment (charlypoly) [inline website/pages/blog/graphql-hive-and-clickhouse.mdx:12]:
```suggestion
You will learn how **ClickHouse** and **Kafka** enabled GraphQL Hive to collect billions of requests monthly without a drop of sweat.
We will also cover the challenges we faced and how we overcame them by covering Hive's architecture in detail.
```

**Comment by charlypoly** [comment]
@kamilkisiela, I did the following:
The Anatomy of a GraphQL Request-1000x400 (5)
caching

**Review by dotansimha — APPROVED** [review]

File: website/pages/blog/graphql-hive-and-clickhouse.mdx [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+## Why ClickHouse?
+
+The kind of data GraphQL Hive deals with is not meant to be mutated (it never changes) and the database has to support a good set of functions for analytics and data aggregation.
+
+Our first choice was **ElasticSearch**. It seemed like a good option, but we hit the first scaling issues very quickly.
+**After reaching the goal of 100M operations per month, the queries to ElasticSearch were unusable.**
+We tried to improve it, but we weren't happy with ElasticSearch anyway.
+
+After looking for alternatives we decided to switch to ClickHouse.
+We highly recommend reading the "Migration from ElasticSearch to ClickHouse" article we wrote with the [ClickHouse Cloud](https://clickhouse.com/cloud) team.
```
Comment (kamilkisiela) [inline website/pages/blog/graphql-hive-and-clickhouse.mdx:70]:
```suggestion
We highly recommend reading the ["100x Faster: GraphQL Hive migration from Elasticsearch to ClickHouse"](https://clickhouse.com/blog/100x-faster-graphql-hive-migration-from-elasticsearch-to-clickhouse) article we wrote with the [ClickHouse Cloud](https://clickhouse.com/cloud) team.
```

**Pull request merged**